node/include/cocaine/detail/service/node/dispatch/handshake.hpp

```diff
@@ -9,11 +9,12 @@
 
 #include "cocaine/idl/rpc.hpp"
 
-#include "cocaine/service/node/slot.hpp"
+#include "cocaine/detail/service/node/forwards.hpp"
```

Comment:
I have the following logic of includes order: from common to details.

1. Standard POSIX or C.
2. Standard C++.
3. Boost.
4. External libraries.
5. Our external libraries.
6. Local libraries.
7. Current library public.
8. Current library details.